randovania/games/cave_story/exporter/options.py

```diff
@@ -3,19 +3,23 @@
 import dataclasses
 from pathlib import Path
 
+from caver.patcher import CSPlatform
+
 from randovania.games.game import RandovaniaGame
 from randovania.interface_common.options import PerGameOptions, decode_if_not_none
 
 
 @dataclasses.dataclass(frozen=True)
 class CSPerGameOptions(PerGameOptions):
-    output_directory: Path | None = None
+    output_directory: Path | None = (None,)
```

Comment:
why is this suddenly a tuple? it doesn't match the type hint and i'm not seeing where it's used as a tuple

Reply:
lemme check, I don't remember changing this manually